src/Backend/Modules/Users/Installer/Installer.php

```diff
@@ -150,20 +150,18 @@ private function loadGodUser(): void
 
             // build settings
             $settings = [];
-            $settings['nickname'] = serialize('Fork CMS');
-            $settings['name'] = serialize('Fork');
-            $settings['surname'] = serialize('CMS');
-            $settings['interface_language'] = serialize($this->getVariable('default_interface_language'));
-            $settings['date_format'] = serialize('j F Y');
-            $settings['time_format'] = serialize('H:i');
-            $settings['datetime_format'] = serialize(
-                unserialize($settings['date_format']) . ' ' . unserialize($settings['time_format'])
-            );
-            $settings['number_format'] = serialize('dot_nothing');
-            $settings['password_key'] = serialize(uniqid('', true));
-            $settings['password_strength'] = serialize($this->getPasswordStrength());
-            $settings['current_password_change'] = serialize(time());
-            $settings['avatar'] = serialize('god.jpg');
+            $settings['nickname'] = 'Fork CMS';
+            $settings['name'] = 'Fork';
+            $settings['surname'] = 'CMS';
+            $settings['interface_language'] = $this->getVariable('default_interface_language');
+            $settings['date_format'] = 'j F Y';
+            $settings['time_format'] = 'H:i';
+            $settings['datetime_format'] = $settings['date_format'] . ' ' . $settings['time_format'];
```

Comment:
The serialisation happens in the model. It unserialized here because a couple of lines above this, it was created and serialised